app/javascript/styles/mastodon/components.scss

```diff
@@ -9008,6 +9008,10 @@ noscript {
     color: $dark-text-color;
     margin-bottom: 20px;
 
+    .inline-block {
+      display: inline-block;
```

Comment:
I *think* `white-space: nowrap` would make more sense, but I am not sure. It seems to behave the same way in this particular case.

Reply:
It didn't seem to work when I tested it previously but I was applying it to the entire P tag in the div and not the specific span. I'll try it there and see.